src/main/resources/swagger/api-docs.yaml

```diff
@@ -412,6 +412,206 @@ paths:
         500:
           description: Internal Server Error
 
+  /api/groups:
+    get:
+      responses:
+        200:
+          description: Successfully listed group membership
+        500:
+          description: FireCloud Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      tags:
+        - Groups
+      summary: get the list of the groups that the caller is a member of
+      operationId: getGroups
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+
+  /api/groups/{groupName}:
+    get:
+      responses:
+        200:
+          description: Successfully loaded group
+        403:
```

Comment:
actually I need to go through and make sure that these status codes are all correct